src/AccordionSection/AccordionHeader.js

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```

Comment:
Most of the `src` changes are just removing the explicit React import since the new transform automatically imports the JSX runtime.  I fixed some minor lint issues that surfaced as well.